lib/src/model/settings/board_preferences.dart

```diff
@@ -300,3 +313,17 @@ enum BoardTheme {
           errorBuilder: (context, o, st) => const SizedBox.shrink(),
         );
 }
+
+enum MaterialDifference {
+  materialDifference(label: 'Material difference', visible: true),
+  capturedPieces(label: 'Captured pieces', visible: true),
+  hidden(label: 'Hidden', visible: false);
+
+  const MaterialDifference({
+    required this.label,
+    required this.visible,
+  });
+
+  final String label;
```

Comment:
The label will be translated at some point, so it's maybe better to already make this a method along the lines of `String l10n(BuildContext context) => switch (this) { ... }` instead (and add a `//TODO l10n`)

Reply:
Sorry, I am not 100% sure what you mean here (though I understand the broader point). Is there an example somewhere else I could look at?

Reply:
Sure, I meant something like this: https://github.com/lichess-org/mobile/blob/ca78193fdd972e9e21ffd282cde7ef2642450d74/lib/src/model/study/study_filter.dart#L16

But actually it's more common in the codebase to do it via a free function, like this: https://github.com/lichess-org/mobile/blob/ca78193fdd972e9e21ffd282cde7ef2642450d74/lib/src/view/settings/theme_screen.dart#L31

Reply:
Hmm yeah I think maybe it sitting in the enum itself makes more sense but I'm not sure. I can implement either approach if there is an opinion either way. For now I might just go with option A, it's easy to change